docs/docs/tutorial/part-6/index.mdx

```diff
@@ -194,21 +194,6 @@ The following diagram shows an overview of the updates you'll have to make in or
 
 ![A diagram showing a new folder structure for the page components, along with the corresponding routes that get generated. Extended description below.](./file-system-routes-with-blog-subdirectory.png)
 
-<Collapsible
```

Comment:
This is a duplicate, the info is related to the paragraphs where the other instance is so remove this one